jkube-kit/jkube-kit-microprofile/src/main/java/org/eclipse/jkube/microprofile/MicroprofileHealthUtil.java

```diff
@@ -31,31 +33,31 @@ public class MicroprofileHealthUtil {
   private MicroprofileHealthUtil() { }
 
   public static boolean hasMicroProfileHealthDependency(JavaProject javaProject) {
-    return JKubeProjectUtil.hasDependency(javaProject, MICROPROFILE_GROUP , MICROPROFILE_HEALTH_DEPENDENCY);
+    return JKubeProjectUtil.hasTransitiveDependency(javaProject, MICROPROFILE_HEALTH_GROUP , MICROPROFILE_HEALTH_DEPENDENCY);
   }
 
   public static boolean hasMicroProfileDependency(JavaProject javaProject) {
-    return JKubeProjectUtil.hasDependency(javaProject, MICROPROFILE_GROUP , MICROPROFILE_DEPENDENCY);
+    return JKubeProjectUtil.hasTransitiveDependency(javaProject, MICROPROFILE_GROUP , MICROPROFILE_DEPENDENCY);
   }
 
   public static boolean isStartupEndpointSupported(JavaProject javaProject) {
     String microProfileHealthVersion;
     if (hasMicroProfileDependency(javaProject)) {
-      microProfileHealthVersion = getMicroProfileVersionFromArtifactId(javaProject, MICROPROFILE_DEPENDENCY);
+      microProfileHealthVersion = getMicroProfileVersionFromArtifactId(javaProject, MICROPROFILE_GROUP, MICROPROFILE_DEPENDENCY);
     } else if (hasMicroProfileHealthDependency(javaProject)) {
-      microProfileHealthVersion = getMicroProfileVersionFromArtifactId(javaProject, MICROPROFILE_HEALTH_DEPENDENCY);
+      microProfileHealthVersion = getMicroProfileVersionFromArtifactId(javaProject, MICROPROFILE_HEALTH_GROUP, MICROPROFILE_HEALTH_DEPENDENCY);
     } else {
       return false;
     }
 
     return isVersionAtLeast(MICROPROFILE_HEALTH_SUPPORTED_VERSION_MAJOR, MICROPROFILE_HEALTH_SUPPORTED_VERSION_MINOR, microProfileHealthVersion);
   }
 
-  private static String getMicroProfileVersionFromArtifactId(JavaProject javaProject, String artifactId) {
-    Dependency microProfileDep = JKubeProjectUtil.getDependency(javaProject, MICROPROFILE_GROUP, artifactId);
+  private static String getMicroProfileVersionFromArtifactId(JavaProject javaProject, String groupId, String artifactId) {
+    Dependency microProfileDep = JKubeProjectUtil.getTransitiveDependency(javaProject, groupId, artifactId);
```

Comment:
Just to double check, this does work with Gradle right? 

(The question is redundant since I see the IT, but it's best if we keep your confirmation to this comment as a record in case there are doubts in the future)

Reply:
I've checked again with my test projet. It seems to be working okay.